package.json

```diff
@@ -10,23 +10,20 @@
     "lint:js:fix": "eslint config src --fix",
     "lint:sass": "stylelint 'src/**/*.scss' --config .stylelintrc.json",
     "nightly": "npm run deploy",
+    "postinstall": "ts-patch install",
+    "release": "semantic-release",
     "start": "fec dev",
     "start:proxy": "PROXY=true fec dev",
     "start:mock": "PROXY=true MOCK=true fec dev",
-    "test:ct": "BABEL_ENV=component cypress run --component",
+    "test": "TZ=UTC jest --verbose --passWithNoTests --no-cache",
+    "test:coverage": "curl -sSL 'https://raw.githubusercontent.com/RedHatInsights/insights-interact-tools/refs/heads/main/scripts/coverage.sh' | bash",
+    "test:ct": "BABEL_ENV=componentTest cypress run --component",
     "test:openct": "BABEL_ENV=component cypress open --component",
-    "test:jest": "TZ=UTC jest --verbose --passWithNoTests --no-cache",
     "test:local": "TZ=UTC jest --verbose --collectCoverage=false",
-    "test": "npm run test:jest",
     "translations": "npm-run-all translations:*",
     "translations:extract": "formatjs extract 'src/**/*.js*' --out-file lang/en.json --id-interpolation-pattern '[sha512:contenthash:base64:6]'",
     "translations:compile": "formatjs compile lang/en.json --out-file compiled-lang/en.json",
-    "verify:local": "npm-run-all build lint test:local test:ct",
-    "verify": "npm-run-all build lint test",
```

Comment:
@opacut issue looks valid, seems ci.int job runs `npm run verify`
```
06:58:58 + npm run verify
06:58:58 npm ERR! Missing script: "verify"
06:58:58 npm ERR! 
```

Reply:
Ohh, then that will mean I need to change that in all the repositories